projects/angular/src/forms/styles/_properties.forms.scss

```diff
@@ -75,7 +75,8 @@
       --clr-forms-radio-disabled-background-color: var(--clr-form-disabled-background-color);
       --clr-forms-radio-disabled-mark-color: var(--clr-forms-checkbox-disabled-mark-color);
       --clr-forms-radio-selected-shadow: var(--clr-forms-checkbox-checked-shadow);
-      --clr-forms-radio-disabled-shadow: var(--clr-forms-checkbox-checked-shadow);
+      --clr-forms-radio-disabled-shadow: #{$clr-forms-radio-disabled-shadow};
```

Comment:
This line was just wrong. It made disabled unchecked radio buttons look the same as non-disabled checked radio buttons.